services/cd-service/pkg/repository/db.go

```diff
@@ -318,3 +429,11 @@ type AllApplicationsGo struct {
 	Created time.Time
 	AllApplicationsJson
 }
+
+type EventRow struct {
+	Uuid       string
+	Timestamp  time.Time
+	CommitHash string
+	EventType  string
```

Comment:
EventType should be an "enum".
See this for an example: https://github.com/freiheit-com/kuberpult/pull/1576/files#diff-8d3024b543096f9dea562737941aa4bd059e0068e3ed46c35c7004ae5f5764f0R205
(This is a different eventType even though the name is the same)